src/Symfony/Component/PasswordHasher/Tests/Hasher/NativePasswordHasherTest.php

```diff
@@ -51,25 +51,25 @@ public function testValidation()
     {
         $hasher = new NativePasswordHasher();
         $result = $hasher->hash('password', null);
-        $this->assertTrue($hasher->verify($result, 'password', null));
-        $this->assertFalse($hasher->verify($result, 'anotherPassword', null));
-        $this->assertFalse($hasher->verify($result, '', null));
+        $this->assertTrue($hasher->verify($result, 'password'));
+        $this->assertFalse($hasher->verify($result, 'anotherPassword'));
+        $this->assertFalse($hasher->verify($result, ''));
```

Comment:
@wouter @chalasr  verify() has no third parameter and hash() has no second salt parameter (anymore?)

Reply:
yes, it's a PasswordEncoderInterface leftover. Not sure about the strategy here but this applies to 5.4 if we want.

Reply:
Done in https://github.com/symfony/symfony/pull/52421